activesupport/lib/active_support/duration/iso8601_parser.rb

```diff
@@ -9,8 +9,6 @@ class Duration
     #
     # This parser allows negative parts to be present in pattern.
     class ISO8601Parser # :nodoc:
-      class ParsingError < ::ArgumentError; end
```

Comment:
On reflection should we even have a custom class - would a simple `ArgumentError` be better?